[Agreements] WHTA.Tamara version 2023.0.8.9

lars-nvhgroup opened the pull request:
cc @denelon @stephengillie

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---


![eula_1](https://github.com/microsoft/winget-pkgs/assets/63705675/2213a4cb-338a-4b56-a4b7-439744134d56)

lars-nvhgroup committed:
WHTA.Tamara version 2023.0.8.9

lars-whta commented:
> **Found duplicate pull request(s)**
> 
> * **Manifest Path: manifests\w\WHTA\Tamara\2023.0.8.9**
>   
>   * [[Agreements] Add back WHTA.Tamara #139463](https://github.com/microsoft/winget-pkgs/pull/139463)

Previous PR should be closed--not sanctioned.

mdanish-kh commented on manifests/w/WHTA/Tamara/2023.0.8.9/WHTA.Tamara.installer.yaml:
```diff
@@ -0,0 +1,39 @@
+InstallerSwitches:
+  Silent: -mode silent
+  SilentWithProgress: -mode silent
+  Custom: -mode silent -agreeToLicense yes
```
```suggestion
  Custom: -agreeToLicense yes
```
Custom represents the args that are passed in addition to the Silent / SilentWithProgress args.

This means with the current manifest, the following args will be passed:

`./installer.exe -mode silent -mode silent -agreeToLicense yes`

We can remove `-mode silent` from `Custom:` to get rid of the duplication

lars-nvhgroup committed:
Update manifests/w/WHTA/Tamara/2023.0.8.9/WHTA.Tamara.installer.yaml

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

lars-nvhgroup committed:
Update manifests/w/WHTA/Tamara/2023.0.8.9/WHTA.Tamara.installer.yaml

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

mdanish-kh commented:
Looks good now, agreements will have to be reviewed by an admin

mdanish-kh commented on manifests/w/WHTA/Tamara/2023.0.8.9/WHTA.Tamara.installer.yaml:
```diff
@@ -0,0 +1,38 @@
+  InstallerSha256: DC10DC695C58672354E687D81E1F8950E7D9A4F35453BFD74C33A8A4F824A25A
+  AppsAndFeaturesEntries:
+  - DisplayName: Tamara
+    Publisher: Wolkesson & Hansson Teknisk Akustik AB
```
```suggestion
    Publisher:
    ProductCode: Tamara
```

Missed this earlier, the installer doesn't write a Publisher value to the registry / Apps&Features / Control Panel. We can use ProductCode to have better package matching

![image](https://github.com/microsoft/winget-pkgs/assets/88161975/d6f9e545-5e28-4735-bd7c-181397ea008c)

lars-nvhgroup committed:
Update manifests/w/WHTA/Tamara/2023.0.8.9/WHTA.Tamara.installer.yaml

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

lars-nvhgroup committed:
Update WHTA.Tamara.installer.yaml

Publisher is updated in most cases, but in some cases it is blank (right after installation). So better to have it pre-filled?
